languages/elixir/exercises/concept/booleans/.meta/config.json

```diff
@@ -0,0 +1,9 @@
+{
+  "authors": [
+    {
+      "github_username": "neenjaw",
+      "exercism_username": "neenjaw"
+    }
+  ],
+  "language_versions": ">=1.10"
```

Comment:
This is for the test runner, as far as I am aware.

Reply:
But even if not, I do think we should encourage students to pick up the latest minor release, because of the docs and whatnot and this is a subtle way to do that.